doc/source/visualizing/plots.rst

```diff
@@ -930,54 +933,148 @@ Use any of the colormaps listed in the :ref:`colormaps` section.
    slc.set_cmap(("gas", "density"), "RdBu_r")
    slc.save()
 
-The :meth:`~yt.visualization.plot_window.AxisAlignedSlicePlot.set_log` function
-accepts a field name and a boolean.  If the boolean is ``True``, the colormap
-for the field will be log scaled.  If it is ``False`` the colormap will be
-linear.
+Colorbar Normalization / Scaling
+::::::::::::::::::::::::::::::::
+
+For a general introduction to the topic of colorbar scaling, see
+`<https://matplotlib.org/stable/tutorials/colors/colormapnorms.html>`_. Here we
+will focus on the defaults, and the ways to customize them, of yt plot classes.
+In this section, "norm" is used as short for "normalization", and is
+interchangeable with "scaling".
+
+Map-like plots e.g., ``SlicePlot``, ``ProjectionPlot`` and ``PhasePlot``,
+default to `logarithmic (log)
+<https://matplotlib.org/stable/tutorials/colors/colormapnorms.html#logarithmic>`_
+normalization when all values are strictly positive, and `symetric log (symlog)
+<https://matplotlib.org/stable/tutorials/colors/colormapnorms.html#symmetric-logarithmic>`_
+otherwise. yt supports two different interfaces to move away from the defaults.
+See **constrained norms** and **arbitrary norm** hereafter.
+
+.. note:: defaults can be configured on a per-field basis, see :ref:`per-field-plotconfig`
+
+**Constrained norms**
+
+The standard way to change colorbar scalings between linear, log, and symmetric
+log (symlog).  Colorbar properties can be constrained via two methods:
+
+- :meth:`~yt.visualization.plot_container.PlotContainer.set_zlim` controls the limits
+ of the colorbar range: ``zmin`` and ``zmax``.
```

Comment:
```suggestion
  of the colorbar range: ``zmin`` and ``zmax``.
```
My bad here.  I forgot the leading whitespace to ensure the indents were correct.  This corrects it.